product_state/views/product_template.xml

```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record model="ir.ui.view" id="product_state_search_form_view">
+        <field name="name">product.state.search.form</field>
+        <field name="model">product.state</field>
```

Comment:
According to the [guidelines](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#15file-naming), the views for model `product.state'` need to be put in a file called `views/product_state_views.xml`, not in `product_template.xml`. Can you move the views, actions and menus for this model to its own xml file?

Reply:
An exception is made for models that are intended to be mostly accessed through a one2many field in the parent model (example, `sale.order.line` going in `sale_order_views.xml` instead of `sale_order_line_views.xml`). Not sure if this is the case, but just for information completeness.

Reply:
Thanks for the clarification, but this model has its own configuration menu, so I think it deserves its own file: 
![image](https://user-images.githubusercontent.com/1033124/204815804-b4c53106-d9ec-4e5f-80bb-477928792700.png)

Reply:
Then yes!

Reply:
Moved to product_state_views.xml